package.json

```diff
@@ -23,6 +23,7 @@
     "@angular/router": "~8.2.14",
     "@ngneat/transloco": "^2.13.5",
     "browserlist": "^1.0.1",
+    "faker": "git+https://github.com/Marak/faker.js.git",
```

Comment:
Lets remove this and make sure that it doesn't make its way into the repo.  This was only meant to be installed in a separate faker project.